### PR TITLE
Updated label value field story, selection card story and switch story

### PR DESCRIPTION
**Pull request by rachana-egov**

### CODE REVIEW

File: flutter/digit-ui-components/storybook/lib/widgets/molecules/list_view_summary_stories.dart [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                height: 50,
+                width: 50,
+                child: Image.network(
+                  'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQIGMLufj86aep95KwMzr3U0QShg7oxdAG8gBPJ9ALIFQ&s',
+                  fit: BoxFit.fill,
+                ),
+              ),
+            ],
+            isInline: isInline, // Set inline property based on knob
+          ));
```
[the remark this answers is not in the document]
Comment (rachana-egov) [reply flutter/digit-ui-components/storybook/lib/widgets/molecules/list_view_summary_stories.dart:90]: this is storybook here they can send anything that is just for showcase

File: flutter/digit-ui-components/storybook/lib/widgets/atoms/list_view_stories.dart [hunk 2]
```diff
@@ -1,86 +1,85 @@
...
-            LabelValuePair(
-              label: context.knobs.text(label: 'Label3', initial: 'label'),
-              value: context.knobs.text(label: 'value3', initial: 'value'),
-              isInline: true,
-            ),
-          ],
+        return Center(
+          child: LabelValueItem(
+            label: label,
+            value: DigitButton(type: DigitButtonType.link,size: DigitButtonSize.small, label: 'Link to profile', onPressed: (){},mainAxisAlignment: MainAxisAlignment.start,),
```
[the remark this answers is not in the document]
Comment (rachana-egov) [reply flutter/digit-ui-components/storybook/lib/widgets/atoms/list_view_stories.dart:80]: it has that parameter

File: flutter/digit-ui-components/storybook/lib/widgets/atoms/selection_card_stories.dart [hunk 3]
```diff
@@ -15,126 +15,142 @@ List<Story> selectionCardStories() {
...
+          // Create an option knob for icons
+          final iconState = context.knobs.nullable.options<String>(
+            label: 'Icon',
+            initial: 'icon', // Ensure an initial value is set
+            options: [
+              Option(label: 'Icon', value: 'icon'),
+              Option(label: 'Icon After Selection', value: 'iconAfterSelection'),
+            ],
+            enabled: false,
+          );
```
[the remark this answers is not in the document]
Comment (rachana-egov) [reply flutter/digit-ui-components/storybook/lib/widgets/atoms/selection_card_stories.dart:99]: i wanted false by default

File: flutter/digit-ui-components/digit_components/lib/widgets/atoms/selection_card.dart [hunk 4]
```diff
@@ -193,7 +186,13 @@ class _SelectionCardState<T> extends State<SelectionCard<T>> {
 
     bool isMobile = AppView.isMobileView(MediaQuery.of(context).size);
 
-    return Column(
+
+    if (widget.equalWidthOptions) {
+      _calculateMaxOptionWidth(context);
+    }
+
```
[the remark this answers is not in the document]
Comment (rachana-egov) [reply flutter/digit-ui-components/digit_components/lib/widgets/atoms/selection_card.dart:193]: i needed context for calculating width that's why

File: flutter/digit-ui-components/storybook/lib/widgets/atoms/selection_card_stories.dart [hunk 5]
```diff
@@ -15,126 +15,142 @@ List<Story> selectionCardStories() {
...
+                      }
+                    }
+                        : null,
+                    errorMessage: errorMessage.isNotEmpty ? errorMessage : null,
+                  );
+                }
+            ),
+          );
+        }
+    ),
```
[the remark this answers is not in the document]
Comment (rachana-egov) [reply flutter/digit-ui-components/storybook/lib/widgets/atoms/selection_card_stories.dart:153]: that is just for story purpose

**Review by naveen-egov — APPROVED** [review]

**Pull request merged**